components/newView.tsx

```
@@ -0,0 +1,198 @@
+            <>
+              {error ? (
+                <div> {error}</div>
+              ) : (
+                <div className="p-6 flex gap-4 flex-col sm:px-24 overflow-y-auto grow">
+                  <h1 className="text-xl font-bold">{adData?.title}</h1>
+                  <div className="text-medium opacity-70 flex flex-col gap-3 z-0">
+                    {adData?.content.map((content, index) => (
+                      <p
+                        key={index}
+                        dangerouslySetInnerHTML={{ __html: content }}
+                      />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)